app/templates/records/new_to_archives.html

```diff
@@ -16,8 +17,114 @@
+            <p>Pellentesque ultrices bibendum nibh, sit amet ornare turpis efficitur id. Aenean ullamcorper neque eget justo sagittis, rutrum ultrices urna varius. Mauris sodales a lorem at sodales.</p>
+          </div>
+        </div>
+        <div class="tna-accordion__item" data-isopen="false">
+          <h3 class="tna-accordion__heading tna-heading-s">
+            What's online
+          </h3>
+          <div class="tna-accordion__body" id="accordion-online">
+            <ul class="tna-ul">
+              <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+              <li>Nulla dignissim vehicula magna, et hendrerit quam iaculis a. Mauris in ultricies enim.</li>
+              <li>Donec bibendum est leo, sed dapibus mauris facilisis vitae.</li>
+              <li>Quisque hendrerit condimentum nisl, non volutpat ex eleifend at. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Morbi id suscipit felis, sed tincidunt arcu. Etiam vel blandit diam, vitae commodo mi.</li>
+            </ul>
+          </div>
+        </div>
+        <div class="tna-accordion__item" data-isopen="false">
+          <h3 class="tna-accordion__heading tna-heading-s">
+            What are archives?
+          </h3>
+          <div class="tna-accordion__body" id="accordion-what-are-archives">
+            <ul class="tna-ul">
+              <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+              <li>Nulla dignissim vehicula magna, et hendrerit quam iaculis a. Mauris in ultricies enim.</li>
+              <li>Donec bibendum est leo, sed dapibus mauris facilisis vitae.</li>
+              <li>Quisque hendrerit condimentum nisl, non volutpat ex eleifend at. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Morbi id suscipit felis, sed tincidunt arcu. Etiam vel blandit diam, vitae commodo mi.</li>
+            </ul>
+          </div>
+        </div>
+        <div class="tna-accordion__item" data-isopen="false">
+          <h3 class="tna-accordion__heading tna-heading-s">
+            How to use archives?
+          </h3>
+          <div class="tna-accordion__body" id="accordion-how-to-use">
+            <ul class="tna-ul">
+              <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+              <li>Nulla dignissim vehicula magna, et hendrerit quam iaculis a. Mauris in ultricies enim.</li>
+              <li>Donec bibendum est leo, sed dapibus mauris facilisis vitae.</li>
+              <li>Quisque hendrerit condimentum nisl, non volutpat ex eleifend at. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Morbi id suscipit felis, sed tincidunt arcu. Etiam vel blandit diam, vitae commodo mi.</li>
+            </ul>
+          </div>
+        </div>
+        
+      </div>
+       <!--end accordion-->
+
+      <!--card/hero element-->
+      <div class="tna-hero tna-!--margin-top-m">
```

Comment:
Ideally, the hero component should sit outside of the container and columns.

I advise you to close off all the parent `<div>`s and use the hero at the top level for this file.

Reply:
I've moved the hero / card element to the upper div level.